chart/etcd-backup-restore/templates/etcd-statefulset.yaml

```diff
@@ -200,6 +200,12 @@ spec:
             secretKeyRef:
               name: {{ .Release.Name }}-etcd-backup
               key: "storageAPIEndpoint"
```

Comment:
An `{{- end }}` statement which is supposed to end the `{{- if .Values.backup.gcs.storageAPIEndpoint }}` block is missing after line 202 - right after `key: "storageAPIEndpoint"` which is cause for the lint error.

Please add this to fix the lint error and thereby fix the build.